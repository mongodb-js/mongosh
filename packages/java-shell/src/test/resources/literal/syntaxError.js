// command printStackTrace
select 1;
