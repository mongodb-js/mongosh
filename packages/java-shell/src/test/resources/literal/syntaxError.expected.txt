java.lang.Exception: SyntaxError: Missing semicolon. (1:6)

> 1 | select 1;
    |       ^
  2 |
    at Parser._raise (all-standalone.js:18680:17)
    at Parser.raiseWithData (all-standalone.js:18673:17)
    at Parser.raise (all-standalone.js:18634:17)
    at Parser.semicolon (all-standalone.js:27791:10)
    at Parser.parseExpressionStatement (all-standalone.js:31019:10)
    at Parser.parseStatementContent (all-standalone.js:30608:19)
    at Parser.parseStatement (all-standalone.js:30472:17)
    at Parser.parseBlockOrModuleBlockBody (all-standalone.js:31061:25)
    at Parser.parseBlockBody (all-standalone.js:31052:10)
    at Parser.parseProgram (all-standalone.js:30393:10)
